packages/langfuse-wrapper/src/langfuse_wrapper/langfuse_wrapper.py

```diff
@@ -0,0 +1,80 @@
+"""
+Langfuse Integration Module
+
+This module provides integration with Langfuse, a tool for monitoring and tracing LLM applications.
+
+Usage:
+    Import this module to enable Langfuse integration.
+    It automatically checks for Langfuse credentials in the .env.langfuse file and for a running Langfuse server.
+    If these are found, it will set up the necessary client and context for tracing.
+
+Note:
+    Run setup_langfuse.sh which automates the steps for running local Langfuse.
+"""
+
+import os
+import logging
+from typing import Callable
+from dotenv import load_dotenv
+from langfuse.decorators import langfuse_context
+import sys
+from io import StringIO
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+if not logger.handlers:
+    handler = logging.StreamHandler()
+    handler.setLevel(logging.INFO)
+    formatter = logging.Formatter("%(asctime)s - %(name)s - %(levelname)s - %(message)s")
+    handler.setFormatter(formatter)
+    logger.addHandler(handler)
+
+CURRENT_DIR = os.path.dirname(os.path.realpath(__file__))
+PACKAGE_ROOT = os.path.dirname(os.path.dirname(os.path.dirname(CURRENT_DIR)))
```

Comment:
this works but i feel we could make this a bit more dynamic using `pathlib` and some marker files

something like this
```python
def find_package_root(start_path: Path, marker_file=('pyproject.toml')) -> Path:
```

if that seems too extra then maybe we can path up explicitly
```python
current_file = Path(__file__).resolve()
package_root = current_file.parents[3]
```